website/src/components/SequenceDetailsPage/DataUseTermsHistoryModal.tsx

```diff
@@ -45,14 +45,14 @@ const DataUseTermsHistoryDialog: FC<DataUseTermsHistoryContainerProps> = ({ data
                 <button className='btn btn-sm btn-circle btn-ghost absolute right-2 top-2'>✕</button>
             </form>
 
-            <h3 className='font-bold text-lg'>Data Use Terms History</h3>
+            <h3 className='font-bold text-lg'>Data use terms history</h3>
 
             <table className='table'>
                 <thead>
                     <tr>
                         <th>Changed</th>
                         <th>User</th>
-                        <th>Data Use Terms</th>
+                        <th>Data use terms</th>
```

Comment:
Should this be changed or do we consider the DUTs as a special name? In the case above, this would mean "Data Use Terms history"

Reply:
I don't see a reason to consider them a special name here - I think one can make more of a case for "Open Data Use Terms" and "Resticted Data Use Terms" being special things as they are specific instances of data use terms.

Reply:
okay, sounds good to me :)